docs/_data/toc.yml

```diff
@@ -41,6 +41,8 @@
     url: guides/multiple-adapters
   - title: Using mesheryctl
     url: guides/mesheryctl
+  - title: Upgrade mesheryctl and Meshery Server
```

Comment:
if only to make the left navigation bar a little more tidy, would it be ok if we updated the title to "Upgrading Meshery"?